src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -81,7 +64,7 @@ public async Task<JsonResult> StartDeployment(string org, string service, string
             if (_configuration["AccessTokenDevOps"] == null)
             {
                 ViewBag.ServiceUnavailable = true;
-                return Json(new
+                return Json(new DeploymentResponse
```

Comment:
Should we change the methods in these methods to return http statuses with models?
So instead it will return Ok(new DeploymentResponse ...)

Reply:
Agreed, makes it cleaner. Fixed :)